ngraph/test/backend/experimental_detectron_roi_feature.in.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// clang-format off
+#ifdef ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#define DEFAULT_FLOAT_TOLERANCE_BITS ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
```

Comment:
Why do we need to have this define?

Reply:
Because it is a backend test, and all backend tests use such define.